packages/discovery-react-components/src/components/SearchResults/components/Result/__tests__/Result.test.tsx

```diff
@@ -752,81 +556,116 @@ describe('<Result />', () => {
             }
-          }
-        ];
-      });
-
-      test('renders result title', () => {
-        const api = {};
-        const { getByText } = render(
-          wrapWithContext(<SearchResults showTablesOnlyToggle />, api, context)
-        );
-        expect(getByText('document title')).toBeInTheDocument();
-      });
-
-      describe('and showOnlyTables is enabled', () => {
-        let renderResult: RenderResult;
-        beforeEach(() => {
-          const api = {};
-          renderResult = render(
-            wrapWithContext(<SearchResults showTablesOnlyToggle />, api, context)
-          );
-          const toggle = renderResult.getByText('Show table results only');
-          fireEvent.click(toggle);
-        });
-
-        test('renders result title', () => {
-          const { getByText } = renderResult;
-          expect(getByText('document title')).toBeInTheDocument();
+          };
         });
 
-        test('renders "View table in document" button', () => {
-          const { getByText } = renderResult;
-          expect(getByText('View table in document')).toBeInTheDocument();
+        test('renders the result title', () => {
+          ({ searchResults } = setup(componentProps, { queryResults, tableResults }));
+          expect(searchResults.getByText('document title')).toBeInTheDocument();
         });
       });
     });
 
-    describe('and there is no corresponding QueryResult', () => {
+    describe('when there are table results', () => {
       beforeEach(() => {
-        (context.searchResponseStore!.data as DiscoveryV2.QueryResponse).results = [
+        tableResults = [
           {
-            document_id: 'some document_id',
-            result_metadata: {
-              collection_id: '123'
-            },
-            document_passages: [
-              {
-                passage_text: 'this is the first passage text'
-              }
-            ]
+            table_id: '558ada041262d5b0aa02a05429d798c7',
+            source_document_id: '123',
+            collection_id: '8713a92b-28aa-b291-0000-016ddc68aa2a',
+            table_html:
+              '<table style="width:100%"><tr><th>Firstname</th><th>Lastname</th><th>Age</th></tr><tr><td>Jane</td><td>Smith</td><td>50</td></tr><tr><td>Eve</td><td>Jackson</td><td>94</td></tr></table>'
           }
         ];
       });
 
-      test('renders the skeleton loading text', () => {
-        const api = {};
-        const { getByTestId, getByText } = render(
-          wrapWithContext(<SearchResults showTablesOnlyToggle />, api, context)
-        );
-        const toggle = getByText('Show table results only');
-        fireEvent.click(toggle);
-        expect(getByTestId('result-title-skeleton')).toBeInTheDocument();
-      });
-
-      describe('and the documents are being fetched', () => {
+      describe("and there is a corresponding QueryResult who's document_id matches the table result source_document_id", () => {
```

Comment:
`who's` --> `whose` :). Also, should we clarify this matching of `document_id` and `source_document_id` in the similar describe block above?

Reply:
good catch this was a duplicate!